pytorch/training/docker/2.1/py3/cu121/Dockerfile.gpu

```diff
@@ -140,6 +140,8 @@ RUN /opt/conda/bin/mamba install -y -c conda-forge \
     pillow \
     "mpi4py>=3.1.4,<3.2" \
     h5py \
+  # dgl is not available in cuda 12+ yet
+  # && /opt/conda/bin/mamba install -y -c dglteam dgl \
```

Comment:
if dglteam is not supported by cuda12+, any reason to add it here?

Reply:
I can just remove it tbh. We can add it later once they support it.